README.md

```diff
@@ -2,8 +2,8 @@
 
 ## Deployments
 
-- Production API: https://leaderboard-serverless.vercel.app/api/
-- staging API: https://staging-leaderboard-serverless.vercel.app/api/
+- Production API: https://leaderboard-serverless.vercel.app/api/  ??
+- staging API: https://staging-leaderboard-serverless.vercel.app/api/  ??
```

Comment:
These links aren't up to date?

Reply:
They are fine. You can execute one of the endpoint, e.g., `https://leaderboard-serverless.vercel.app/api/aggregated_stats` and it'll give you the data.

About the staging, we could remove it, because we don't run orch-tester in staging.

Reply:
Oh! that's just API. I thought this `404` means they don't work at all